main/options/options.cc

```diff
@@ -598,10 +600,16 @@ void readOptions(Options &opts, int argc, char *argv[],
         }
         opts.disableWatchman = raw["disable-watchman"].as<bool>();
         opts.watchmanPath = raw["watchman-path"].as<string>();
+
+        // Certain features only need certain passes
         if ((opts.print.Autogen.enabled || opts.print.AutogenMsgPack.enabled || opts.print.AutogenClasslist.enabled) &&
-            (opts.stopAfterPhase != Phase::NAMER)) {
+            (opts.stopAfterPhase < Phase::NAMER)) {
             logger->error("-p autogen{} must also include --stop-after=namer",
-                          opts.print.AutogenMsgPack.enabled ? "-msgpack" : "");
+                          opts.print.AutogenMsgPack.enabled ? "-msgpack" : ""); // TODO(gwu) Fix this
```

Comment:
Ah forgot about this TODO—since this is pay-server specific (and the flag will probably always be programmatically supplied) do you want me to fix this so it doesn't always just say `-msgpack must also include...`?